src/ServiceInsight/App.xaml

```diff
@@ -59,8 +59,6 @@
                         <GradientStop Offset="0" Color="#FFFFFAE9" />
                         <GradientStop Offset="1" Color="#FFFFE8A6" />
                     </LinearGradientBrush>
-                    <SolidColorBrush x:Key="FocusedRowBackgroundBrush" Color="#222222" />
-                    <SolidColorBrush x:Key="FocusedRowForegroundBrush" Color="White" />
```

Comment:
As @WojcikMike correctly points out in the comment below this section needs to stay otherwise the trigger

```
<DataTrigger Binding="{Binding Path=IsSelected}" Value="True">
```

cannot set the expected values.